power/Power.cc

```diff
@@ -1102,7 +1103,9 @@ Power::findOutputInternalPower(const LibertyPort *to_port,
   FuncExpr *func = to_port->function();
 
   map<const char*, float, StringLessIf> pg_duty_sum;
+  int numArcs = 0;
   for (InternalPower *pwr : corner_cell->internalPowers(to_corner_port)) {
+    numArcs += 1;
```

Comment:
the variable does not follow the coding guidelines.
look around. you will never see "num" anything.
It should be arc_count